client/jetpack-cloud/sections/partner-portal/license-list/empty.tsx

```diff
@@ -27,9 +30,7 @@ export default function LicenseListEmpty( { filter }: Props ) {
 	const licenseFilterStatusTitle = licenseFilterStatusTitleMap[ filter ] as string;
 
 	const onIssueNewLicense = () => {
-		dispatch(
-			recordTracksEvent( 'calypso_partner_portal_license_list_empty_issue_license_click' )
-		);
+		dispatch( recordTracksEvent( 'calypso_partner_portal_issue_license_click_billing_page' ) );
```

Comment:
Could you explain what's the reason for this event name change?

I see two issues arising from it:

1. Attempting to understand the usage of the button will be difficult since the event name will change after we merge this. For example, if I create Tracks Funnel to understand the use of the button, I would have to select one event name or the other, essentially choosing what happened before or after the name change.
2. There are naming conventions we need to follow to name events. The proposed name doesn't follow them since the action (`click`) is not the final part of the string. See PCYsg-4S2-p2.

Reply:
> Could you explain what's the reason for this event name change?

No reason to change it. It's a mistake copying the name from the previous commit where I had more lines in that function. So good catch here :)

Reply:
Fixed here: https://github.com/Automattic/wp-calypso/pull/84188/commits/c53197547d4c20f182cbc481508478b55e9b2304